libazureinit/src/http.rs

```diff
@@ -111,7 +111,7 @@ async fn request(
         let now = std::time::Instant::now();
         let mut attempt =  0_u32;
         loop {
-            let span = tracing::info_span!("request", attempt, http_status = tracing::field::Empty);
+            let span = tracing::debug_span!("request", attempt, http_status = tracing::field::Empty);
```

Comment:
I could go either way here, but I lean towards this being info-level (see my rambling comment on log levels below). I don't feel terribly strongly about it, though.